lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -257,6 +257,7 @@ class MailboxDashBoardController extends ReloadableController {
     if (arguments is Session) {
       sessionCurrent = arguments;
       accountId.value = sessionCurrent?.accounts.keys.first;
+      injectAutoCompleteBindings();
```

Comment:
I though, these components are injected when initialized the app? why we need this after getSession and reloaded ?

Reply:
It will not be included when launching the application. We need to check if the current server supports `autocomplete` so we need to initialize it when we get the `session`.

Reply:
sorry, I forgot it :)